api/src/pcapi/core/bookings/models.py

```diff
@@ -260,6 +260,16 @@ def userName(self) -> str:
     def email(self) -> str:
         return self.user.email
 
+    @property
+    def emailAtDateCreated(self) -> str:
+        email = self.user.email
+        for history in reversed(sorted(self.user.email_history, key=lambda history: history.creationDate)):
+            print(history.id, history.creationDate, self.id)
+            if self.dateCreated >= history.creationDate:
+                email = f"{history.newUserEmail}{history.newDomainEmail}"
```

Comment:
```suggestion
                email = history.newEmail
```
Il manque l'arobase sur ta ligne, mais utilise plutôt directement la propriété `newEmail` qui fait la concaténation.

Reply:
A priori non : 
https://github.com/pass-culture/pass-culture-main/blob/v231.0.5/api/src/pcapi/core/users/models.py#L718